home02/js/script_7.js

```diff
@@ -0,0 +1,14 @@
+// Запросить у пользователя сумму покупки и вывести сумму к оплате со скидкой: от 200 до 300 – скидка будет 3%, от 300 до 500 – 5%, от 500 и выше – 7%.
+
+const sumPurchase = +prompt(`What is the amount of purchases`);
+
+if (200 <= sumPurchase && sumPurchase <= 300) {
+    alert(`The amount of purchases with a 3% discount: ${sumPurchase - (sumPurchase * 0, 03)}`);
```

Comment:
дробные числа - через точку
```suggestion
    alert(`The amount of purchases with a 3% discount: ${sumPurchase - (sumPurchase * 0.03)}`);
```